src/common.h

```diff
@@ -3,6 +3,7 @@
 
 #define PROXYCHAINS_CONF_FILE_ENV_VAR "PROXYCHAINS_CONF_FILE"
 #define PROXYCHAINS_QUIET_MODE_ENV_VAR "PROXYCHAINS_QUIET_MODE"
+#define PROXYCHAINS_CONFIG_DATA_ENV_VAR "PROXYCHAINS_CONFIG_DATA"
```

Comment:
i told you to re-use the PROXYCHAINS_CONF_FILE env var rather than using a new one. just use the magic prefix fd: (or maybe better fd:// so it's invalid as a file name)